apps/beeswax/src/beeswax/tests.py

```diff
@@ -1762,7 +1762,8 @@ def test_select_query_server(self):
 
```

Comment:
Would you know the value of `HIVE_SERVER_HOST.get()` before sending the query? Feels like a bug as we should not need to to the change below but my guess is that this code path is not used anymore